packages/standard-server-peer/src/server.test.ts

```
@@ -469,31 +590,50 @@ describe('serverPeer', () => {
           finally {
             isFinallyCalled = true
             // eslint-disable-next-line no-unsafe-finally
-            throw new Error('should silence ignored')
+            throw new Error('cleanup error')
```

Biome (2.1.2)

[error] Unsafe usage of 'throw'.

'throw' in 'finally' overwrites the control flow statements inside 'try' and 'catch'.

(lint/correctness/noUnsafeFinally)